File: grabber/src/main/kotlin/com/github/pozo/investmentfunds/grabber/InvestmentFundsApplication.java
package com.github.pozo.investmentfunds.grabber;

import org.apache.camel.main.Main;

/**
 * Main class that boot the Camel application
 */
public final class InvestmentFundsApplication {

    private InvestmentFundsApplication() { }

    public static void main(String[] args) throws Exception {
        // use Camels Main class
        Main main = new Main(InvestmentFundsApplication.class);
        // now keep the application running until the JVM is terminated (ctrl + c or sigterm)
        main.run(args);
    }

}

File: api/src/main/kotlin/com/github/pozo/investmentfunds/api/InvestmentFundsApplication.kt
package com.github.pozo.investmentfunds.api

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication
open class InvestmentFundsApplication

fun main(args: Array<String>) {
    runApplication<InvestmentFundsApplication>(*args)
}
